### ISSUE TITLE
Implement missing properties and methods in NSPrintOperation

### ISSUE THREAD

**Issue by CKegel** [darlinghq/darling#1485]
Currently our implementation of [NSPrintOperation](https://developer.apple.com/documentation/appkit/nsprintoperation?language=objc) in the Cocotron is missing several selectors and properties - most notably `jobTitle`.

**Comment by Lazerbeak12345** [comment]
Do you have a good way to test these APIs?

**Comment by CKegel** [comment]
Yes, these operations are used when printing documents in TextEdit
